### ISSUE TITLE
After pressing next/previous button focus remains on those buttons and no announcement is made by screen reader for the updated content.

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4593]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.3 Status Messages

**Test Environment:**
- Test URL: https://clarity.design/documentation/alerts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to previous or next button provided for Multiple app level alerts and activate it by pressing enter or space key.

**Expected result:**
If the user presses either previous or next button. Appropriate announcement should be made by screen reader to notify user about the onscreen change. It can be something like &quot;Showing alert 2 of 3&quot;.

**Actual result:**
No announcement is made for onscreen changes of the alert message.

**Proposed solution:**
Use Aria-live region to track changes in the message and notify users about the changes. Reference link: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Reporter Email:** gkhadke@vmware.com

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
